flake.nix

```diff
@@ -0,0 +1,32 @@
+{
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    naersk.url = "github:nix-community/naersk";
```

Comment:
Hi! from your perspective, how does Naersk compare with Crane (if you've used both?) in terms of UX & philosophy? We're familiar with Crane but not with naersk as of present. We do some Nix building internally and so ideally would become most familiar with one Rust build tool.

Reply:
I haven't used crane. Naersk is recommended by the NixOS wiki and that's the only thing I tried.

I tried it on a more complicated repo, with multiple rust modules.